src/celestia/qt/qtappwin.cpp

```diff
@@ -1084,12 +1085,13 @@ void CelestiaAppWindow::slotBookmarkTriggered(const QString& url)
 
 void CelestiaAppWindow::slotManual()
 {
-#if 0
-    QString MANUAL_FILE = "CelestiaGuide.html";
-    QDesktopServices::openUrl(QUrl(QUrl::fromLocalFile(QDir::toNativeSeparators(QApplication::applicationDirPath()) + QDir::toNativeSeparators(QDir::separator()) + "help" + QDir::toNativeSeparators(QDir::separator()) + MANUAL_FILE)));
-#else
+    QDesktopServices::openUrl(QUrl::fromLocalFile(QDir::toNativeSeparators(m_dataHome) + QDir::toNativeSeparators("/help/CelestiaGuide.html")));
```

Comment:
```suggestion
    QDesktopServices::openUrl(QUrl::fromLocalFile(QDir::toNativeSeparators(m_dataHome + "/help/CelestiaGuide.html")));
```

Reply:
Tried and toNativeSeparators will not perform successfully on any mixed directory styles.  This code separates path segments so if some combination of styles should get into the input it is correctly resolved. A single string will fail in this condition.  It is preventing a possible condition when so many operating systems are involved and cannot be tested on all potentials.